docs/_docs/rql.md

```diff
@@ -0,0 +1,470 @@
+---
+title: Resource Query Language
+---
+
+* [Background](#background)
+* [AST Grammar](#ast-grammar)
+* [Primaries](#primaries)
+  * [action](#action)
+  * [boolean](#boolean)
+  * [name](#name)
+  * [cname](#cname)
+  * [path](#path)
+  * [kind](#kind)
+  * [atime](#atime)
+  * [crtime](#crtime)
+  * [ctime](#ctime)
+  * [mtime](#mtime)
+  * [size](#size)
+  * [meta](#meta)
+* [Detailed Meta Primary Overview](#detailed-meta-primary-overview)
+
+## Background
+
+The resource query language (RQL) lets you query Wash resources. You can submit your queries to the API's `find` endpoint. Here's an example query
+
+```
+Eniss-MacBook-Pro:wash enisinan$ curl -X POST --unix-socket /tmp/WASH_SOCKET --header "Content-Type: application/json" --data '["kind", ["glob", "*ec2*instance"]]' 'http://localhost:/fs/find?path=/tmp/WASH_MOUNT/aws/wash' 2>/dev/null | jq
```

Comment:
I'd simplify the prompt to
```suggestion
$ curl -X POST --unix-socket /tmp/WASH_SOCKET --header "Content-Type: application/json" --data '["kind", ["glob", "*ec2*instance"]]' 'http://localhost/fs/find?path=/tmp/WASH_MOUNT/aws/wash' 2>/dev/null | jq
```

Also, what were you getting on stderr?

Reply:
Ok, can do that.

stderr I was getting progress status.